# 以RPC调用的形式获取当前传感器的数值
---
float32 force_x
float32 force_y
float32 force_z
float32 moment_x
float32 moment_y
float32 moment_z